action_plugins/tasmota.py

```diff
@@ -185,11 +188,13 @@ def our_entry(x):
         display.v("[%s] existing_uri: %s" % (tasmota_host, endpoint_uri))
         if existing_value != incoming_value:
             changed = True
-            change_params = copy.deepcopy(auth_params)
+
+            if not check_mode:
+                change_params = copy.deepcopy(auth_params)
             change_params.update( { 'cmnd' : ("%s %s" % (command, incoming_value)) } )
```

Comment:
@cwendel16834  thanks for your contribution. This line looks like there is a indent missing or am I wrong?

Reply:
Great catch! I'll clean that up.

Reply:
Fixed in next commit.

Reply:
Thanks!